ts_benchmark/pipeline.py

```diff
@@ -156,10 +155,12 @@ def pipeline(
         model_factory_list, result_list, model_save_names
     ):
         for i, result_df in enumerate(result_itr.collect()):
+            if not evaluation_config.get("save_true_pred", False):
+                result_df["actual_data"], result_df["inference_data"] = np.nan, np.nan
```

Comment:
it is not a good idea to determine here whether to save these data, because encoding these data could be time consuming and should be avoided if they are not to be saved.

Reply:
It has been modified.